pennylane_lightning/core/src/simulators/lightning_qubit/measurements/MeasurementKernels.hpp

```diff
@@ -296,6 +297,86 @@ namespace PUtil = Pennylane::Util;
 
 namespace Pennylane::LightningQubit::Measures {
 
+/**
+ * @brief Generate samples using the alias method.
+ *
+ * @tparam PrecisionT Precision data type
+ */
+template <typename PrecisionT> class discrete_random_variable {
```

Comment:
Could you update the name of this class to follow the style?
```suggestion
template <typename PrecisionT> class DiscreteRandomVariable {
```